scanner/jsFramework.go

```diff
@@ -198,6 +202,9 @@ func configureJsFramework(sourceDir string, config *ScannerConfig) (*SourceInfo,
 	} else if deps["gatsby"] != nil {
 		srcInfo.Family = "Gatsby"
 		srcInfo.Port = 8080
+	} else if scripts["dev"] == "vite" {
```

Comment:
Move this after remix?

Reply:
oops you're right, my brain was thinking of the last change i made where it was a bunch of `if`s and not `else if`s